prql-lib/README.md

```diff
@@ -0,0 +1,81 @@
+# PRQL library target
+
+## Description
+This module compiles PRQL as a library (both `.a` and `.so` are generated). This allows embedding in languages that support FFI - looking at Golang.
+
+## Usage
+
+Copy the `.a` and `.so` files in a convenient place and add the following compile flags to Go (cgo):
+
+`CGO_LDFLAGS="-L/path/to/libprql_lib.a -lprql_lib -pthread -ldl" go build`
+
+## Code
+
+Below is an example from an actual application that is using PRQL in Go.
+
+```
```

Comment:
```suggestion
```go
```

Reply:
I've pushed a commit with this change. Thanks